api/src/main/java/com/datatorrent/api/ControlTupleEnabledSink.java

```diff
@@ -18,7 +18,7 @@
  */
 package com.datatorrent.api;
 
-import org.apache.apex.api.UserDefinedControlTuple;
+import org.apache.apex.api.ControlTuple;
```

Comment:
Operator developers would still be our users but not necessarily platform developers. I am leaning towards keeping it straightforward and simple for our users. We could use more descriptive names for the ones used by the platform.